### PR TITLE
add project admin

### PR DESCRIPTION
**Pull request by fiennyangeln**
#3

### CODE REVIEW

**Comment by madprime** [comment]
I see a couple issues with this PR... I think I'm likely to merge #11 instead, but I'm going to leave comments to help teach stuff. Hopefully this helps!

File: project_admin/templates/oh_proj_management/base.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{% load static %}
```
Comment (madprime) [inline project_admin/templates/oh_proj_management/base.html]: (This comment is related to this whole file, not this particular first line.)

I'd rather see this file to have been created by moving the original file to this location and name (using `git mv`) not making a copy of it. The old location should be obsolete (replaced by the new location), and by using `git mv` you retain authorship history for items within the file.

File: project_admin/templates/oh_proj_management/home.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{% extends 'oh_proj_management/base.html' %}
```
Comment (madprime) [inline project_admin/templates/oh_proj_management/home.html]: When moving a templates to a new app, I'd prefer the path for the templates follow the new app's naming convention, for example: `myapp/templates/myapp/foo.html`

**Comment by fiennyangeln** [comment]
Ok so do i need to work on this / is it tackled already?

**Comment by gedankenstuecke** [comment]
This has been solved with #11. But did you find the comments of @madprime helpful for further work? :)

**Pull request closed without merging**